corehq/util/context_processors.py

```diff
@@ -266,3 +266,9 @@ def get_demo(request):
             'is_demo_visible': True,
         })
     return context
+
+
+def status_page(request):
+    return{
```

Comment:
Odd that stickler didn't prompt you to put a space after return, but this needs one